lib/pengine/pe_digest.c

```diff
@@ -331,29 +349,40 @@ pe__calculate_digests(pe_resource_t *rsc, const char *task, const char *key,
  * \return Pointer to node's digest cache entry
  */
 static op_digest_cache_t *
-rsc_action_digest(pe_resource_t *rsc, const char *task, const char *key,
+rsc_action_digest(pe_resource_t *rsc, const char *task, guint interval_ms,
                   pe_node_t *node, xmlNode *xml_op, bool calc_secure,
                   pe_working_set_t *data_set)
 {
     op_digest_cache_t *data = NULL;
+    char *key = pcmk__op_key(rsc->id, task, interval_ms);
```

Comment:
shouldn't the key be calculated after calling pe__calculated_digests that potentially changes interval_ms?

Reply:
aah no ... sorry the cache is searched with the initial key